frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/space/sidebar_space_header.dart

```diff
@@ -39,73 +38,68 @@ class SidebarSpaceHeader extends StatefulWidget {
 }
 
 class _SidebarSpaceHeaderState extends State<SidebarSpaceHeader> {
-  final isHovered = ValueNotifier(false);
+  final _isHovered = ValueNotifier(false);
```

Comment:
IMO, if the state is private, it's not necessary to use underscores for properties defined inside it. :)

Reply:
That's a good point. I'll rename the value in the valuelistenablebuilder to something else